### DISCUSSION TITLE
Flatten middle packages

### DISCUSSION THREAD

**Discussion by WildOrangutan** [community/community#101740] [Mobile] [answered]
When browsing code on a mobile app, some code often has directories like `java/com/some/app`, with no files in between, only directories.

Currently, we have to open each directory 1 by 1, which is annoying.

It would be nice to flatten directories, so you navigate directly to the last directory, e.g. `app`.

**Comment by stevepopovich** [comment, marked as answer]
Hi @WildOrangutan yep, this could be nice to have. I will pass this on to the team for internal tracking. Thanks for writing in!
